11/umbraco-forms/editor/creating-a-form/creating-a-contact-us-form.md

```diff
@@ -0,0 +1,265 @@
+21. Select **How should we contact you from the dropwdown**.
+22. Enter **phone** in the value field.
+23. Click **Submit**.
+24. Click the **Add Question** button to add the next question.
+25. Enter the following details:
+
+    | Field Name         | Value                        |
+    |--------------------|------------------------------|
+    | Enter question     | Enter your email address     |
+    | Choose answer type | Short answer                 |
+    | Field Type         | email                        |
+    | Validation         | Validate as an email address |
+
+26. Select **Enable Conditions**.
+27. Click **Add Condition**.
+28. Select **How should we contact you from the dropwdown**.
+29. Enter **email** in the value field.
+30. Click **Submit**.
+31. Click the **Add Question** button to add the next question.
+32. Enter the following details:
+
+    | Field Name         | Value                                                    |
+    |--------------------|----------------------------------------------------------|
+    | Enter question     | What is your role?                                       |
+    | Choose answer type | Dropdown                                                 |
+    | Prevalues Items    | manager, developer, tester, writer, marketing specialist |
+
+33. Click **Submit**.
+34. Click the **Add Question** button to add the next question.
+35. Enter the following details:
+
+    | Field Name                     | Value                    |
+    |--------------------------------|--------------------------|
+    | Enter question                 | Attachments (if any)     |
+    | Choose answer type             | File upload              |
+    | Predefined allowed file types  | pdf, png, jpg, gif, txt  |
+
+36. Click **Submit**.
+37. Click the **Add Question** button to add the next question.
+38. Enter the following details:
+
+    | Field Name         | Value               |
+    |--------------------|---------------------|
+    | Enter question     | Are you a Robot?    |
+    | Choose answer type | reCAPTCHAv2         |
+    | Theme              | light               |
+    | Size               | normal              |
+    | Mandatory          | Yes                 |
+
+39. Click **Submit**.
+
+    ![Add questions](images/contact-us-form-add-questions.png)
+40. Click **Reorder**.
+41. Drag the **Data consent** group below the **Information** group.
+42. Click **I am done reordering**.
+43. Click **Save**.
+
+## Configure Workflow
+
+To configure the form workflow, follow these steps:
+
+1. Click on **Submit message/ Go to page**.
+2. Enter a customized message in the **Message on Submit** field.
+3. Click **Submit**.
+4. Click on **Send template email to xxx@xx.dk**.
+5. Select **Example-Template.cshtml** in the **Email template** field.
+6. Enable **Attachments**.
+7. Enter an email address in the **Sender Email** field.
+8. Click **Submit**.
+9. Click **Save**.
+
+## Configure Form Settings
```

Comment:
Are **all** these steps necessary?
I don't think you need to provide a stylesheet, or enter a custom label for the Submit, Next and Previous buttons fx.

Reply:
No, all these steps are not required. In the video, I've only done 1 step ie. Select **Mark Mandatory fields** in the **Mark fields**. I can add just 1 step but it looks weird so I added the rest of the options as well. 

I can mention though that these steps are optional and not mandatory.